docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -51,12 +51,12 @@ To successfully deploy, the {ProductShortName} Operator requires 3 RWO persisten
 
 == Installing the {ProductName} Operator and the {WebName}
 
-You can install the {ProductName} ({ProductShortName}) and the {WebName} on OpenShift Container Platform versions 4.9-4.12 when you install the {ProductName} Operator.
+You can install the {ProductName} ({ProductShortName}) and the {WebName} on OpenShift Container Platform versions 4.10-4.14 when you install the {ProductName} Operator.
```

Comment:
@anarnold97 
It's an awkward phrase. Why not "You can install the {ProductName} ({ProductShortName}) Operator and the {ProductShortName} {WebName} on OpenShift Container Platform versions 4.10-4.14. "?